src/commonMain/kotlin/fr/acinq/lightning/channel/WaitForAcceptChannel.kt

```diff
@@ -79,6 +117,33 @@ data class WaitForAcceptChannel(
         }
     }
 
+    private fun createFundingTx(fundingPubkeyScript: ByteVector): Pair<Transaction, Satoshi> {
+        val txIn = init.fundingInputs.inputs.map { i -> TxIn(i.outpoint, 0) }
```

Comment:
nit: `txIn` is a collection, it should be a plural like `inputs`

Reply:
Fixed in https://github.com/ACINQ/lightning-kmp/pull/365/commits/79e69505c70b9c4ffd1f3795ee071bf77ca28296